requirements/requirements-ci.txt

```diff
@@ -121,7 +121,7 @@ pycryptodome==3.8.2
 pyflakes==2.1.1
 pygments==2.4.2
 pyhamcrest==1.9.0
-pyinstaller==3.4
+pyinstaller==3.6
```

Comment:
This pinning should go into `requirements-ci.in` and then run `requirements/deps upgrade pyinstaller`

Reply:
Ah damn. Thanks. To be fair I knew there was something like that after reading the readme but I tried `requirements/deps upgrade pyinstaller==3.6 ` and got a failure. Let me do that then.

Reply:
done